fe/fe-core/src/test/java/org/apache/doris/analysis/ColumnDefTest.java

```diff
@@ -133,7 +134,7 @@ public void testBooleanDefaultValue() throws AnalysisException {
 
     @Test
     public void testArray() throws AnalysisException {
-        ctx.getSessionVariable().setEnableArrayType(true);
+        Config.enable_array_type = true;
```

Comment:
MayBe it is more reasonable to put this code in setUp.

Reply:
done